tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+    def _load_config(self):
+        """Load configuration from YAML file"""
+        if not self.config_path.exists():
+            raise FileNotFoundError(f"Configuration file not found: {self.config_path}")
+
+        with open(self.config_path, "r") as f:
+            raw_config = yaml.safe_load(f)
+
+        # Expand environment variables
+        self._config = self._expand_env_vars(raw_config)
+
+    def _expand_env_vars(self, obj):
```

Ruff (0.11.9)

Missing return type annotation for private function `_expand_env_vars`

(ANN202)